lib/static/components/icons/view-in-browser/index.jsx

```diff
@@ -41,7 +41,7 @@ class ViewInBrowser extends Component {
         return (
             <a
                 className={className}
-                href={getUrlWithBase(getRelativeUrl(suiteUrl), baseHost)}
+                href={getUrlWithBase(suiteUrl, baseHost)}
```

Comment:
what was the problem with using `getRelativeUrl` method?

Reply:
in case of empty baseHost relative path will be returned:

```
getUrlWithBase(getRelativeUrl("https://ya.ru/some/path"), null); ==> /some/path
```

and such relative paths will be resolved in the browser relative to the report host